tools/accuracy_checker/configs/yolo-tiny-v2-ava-0001.yml

```diff
@@ -0,0 +1,46 @@
+models:
+  - name: yolo-tiny-v2-ava-0001
+
+    launchers:
+      - framework: dlsdk
+        tags:
+          - FP32
+        model: intel/yolo-tiny-v2-ava-0001/FP32/yolo-tiny-v2-ava-0001.xml
+        weights: intel/yolo-tiny-v2-ava-0001/FP32/yolo-tiny-v2-ava-0001.bin
+        adapter:
+          type: yolo_v2
+          anchors: tiny_yolo_v2
+      - framework: dlsdk
```

Comment:
please also add launcher for FP16 for all 3 models

Reply:
fixed